ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/lldp.j2

```diff
@@ -16,4 +19,37 @@ lldp management-address {{ lldp.management_address }}
 {%     if lldp.vrf is arista.avd.defined %}
 lldp management-address vrf {{ lldp.vrf }}
 {%     endif %}
+{%     if lldp.receive_packet_tagged_drop is arista.avd.defined(true) %}
+lldp receive packet tagged drop
+{%     endif %}
+{%     if lldp.tlv_transmit.link_aggregation is arista.avd.defined(false) %}
```

Comment:
for each tlv option, we should check both true and false and output accordingly. Some of the options are disabled by default in EOS, and we should not maintain this in AVD.

Reply:
Hi Claus,
I was just checking this. On a 4.24.3M and belowl I can see that all TLVs are eneabled by default:
```
sh run all | i lldp
lldp timer 30
lldp hold-time 120
lldp timer reinitialization 2
lldp tlv transmit link-aggregation
lldp tlv transmit management-address
lldp tlv transmit max-frame-size
lldp tlv transmit port-description
lldp tlv transmit port-vlan
lldp tlv transmit power-via-mdi
lldp tlv transmit system-capabilities
lldp tlv transmit system-description
lldp tlv transmit system-name
lldp tlv transmit med network-policy
lldp run
```

on a 4.26.3M one additional tlv was added and is disabled by default:

```
sh run all | i lldp
lldp timer 30
lldp hold-time 120
lldp timer reinitialization 2
lldp tlv transmit link-aggregation
lldp tlv transmit management-address
lldp tlv transmit max-frame-size
lldp tlv transmit port-description
lldp tlv transmit port-vlan
lldp tlv transmit power-via-mdi
lldp tlv transmit system-capabilities
lldp tlv transmit system-description
lldp tlv transmit system-name
**no lldp tlv transmit vlan-name**
lldp tlv transmit med network-policy
lldp run
```

are you referring to this one? because I havent even covered support for that one yet, since it wasn't present in the older EOS.

Reply:
I think this just proves that we need to solve this in a different way.
So I suggest to change the data model to:
```yaml
lldp.transmit_tlvs:
  - name: "<TLV name>"
    transmit: < true | false >
```
Example:
```yaml
lldp.transmit_tlvs:
  - name: "link-aggregation"
    transmit: true
​```

So when we output the CLI, we just take whatever is in the `name` var and add it to `lldp tlv transmit` with or without a `no ` before, depending on the `transmit`​ flag.

This will make this maintenance free :)

Reply:
makes sense! i will update it in a bit